src/main/java/core/basesyntax/Shape.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public abstract class Shape implements Figure {
+    private double area;
+    private String color;
+
+    public Shape(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public double getArea() {
```

Comment:
you override this method in each class, so let's create a new interface and move this method there, and then remove field area from the class